lib/Graph/Nodes.cpp

```diff
@@ -1772,22 +1773,13 @@ bool ResizeNearestNode::verify() const {
                                size_t(4), this);
   isValid &= expectCompareTrue("Output must be a 4D tensor", outputDims.size(),
                                size_t(4), this);
-  isValid &= expectCompareTrue("Batch size must be the same", inputDims[0],
-                               outputDims[0], this);
-  isValid &= expectCompareTrue("Depth must be the same", inputDims[3],
-                               outputDims[0], this);
-  isValid &= expectCompareTrue(
-      "Unexpected output height",
-      dim_t(std::floor(inputDims[1] * getHeightScale())), outputDims[1], this);
-  isValid &= expectCompareTrue(
-      "Unexpected output width",
-      dim_t(std::floor(inputDims[2] * getWidthScale())), outputDims[2], this);
-  isValid &=
-      expectCompareTrue("Invalid height scale", getHeightScale(), float(0.0),
-                        this, CompareOperatorGreaterThan<float>());
-  isValid &=
-      expectCompareTrue("Invalid width scale", getWidthScale(), float(0.0),
-                        this, CompareOperatorGreaterThan<float>());
+  for (size_t i = 0; i < scale.size(); i++) {
```

Comment:
nit: `  for (size_t i = 0, e = scale.size(); i < e; i++)`

Reply:
Done